src/gateways/createBuyPrintConfigLoader.ts

```diff
@@ -18,6 +19,17 @@ export default (url: string) => {
         } else {
           return rejectResponse(response,  (status, message) => `Error response from BuyPrint ${status}: ${message}`);
         }
+      }).catch((e) => {
+        Sentry.captureException(e);
+
+        return Promise.resolve({
+          buy_urls: [{
+            allows_redirects: false,
```

Comment:
it returns `true` for https://buyprint.openstax.org/physcis.json
I think it should be `true` in this case since we will be redirecting by default

Reply:
yea this field is irrelevant but i think `true` would be the correct value